packages/web-console/src/scenes/Editor/Monaco/index.tsx

```diff
@@ -229,6 +230,20 @@ const MonacoEditor = () => {
     }
   }
 
+  const removeErrors = (
+    editor: IStandaloneCodeEditor,
+    monaco: Monaco,
+  ) => {
+    const model = editor.getModel()
+    if (model) {
+      monaco.editor.setModelMarkers(model, QuestDBLanguageName, []);
+    }
+
+    errorRangeRef.current = undefined
+    errorRef.current = undefined
+    dispatch(actions.query.cleanupNotifications())
```

Comment:
i think we can continue with what i wrote in [this comment](https://github.com/questdb/ui/pull/134/files#r1183620021) but then this line should not be here.

After user does a change on an erroneous query, this does not guarantee that the change is a fix. Removing error range marker and the notification cleans all info. I think notification should remain.